README.md

```diff
@@ -47,16 +47,14 @@ linter after my tests I would do so like this.
 [dogma]: https://github.com/lpil/dogma
 
 ```elixir
-# config/config.exs
+# config/dev.exs
```

Comment:
The default config.exs of a new Elixir project (`mix new myapp`) has comments that suggest creating separate config files. I assumed any project of reasonable size would have dev.exs. You want to encourage using this way of working, instead of adding `if` blocks in config.exs?